src/python/pants/option/config_test.py

```diff
@@ -22,72 +38,175 @@ def setUp(self) -> None:
       disclaimer:
         Let it be known
         that.
-      blank_section:
 
       [a]
       list: [1, 2, 3, %(answer)s]
-      listappend: +[7, 8, 9]
+      list2: +[7, 8, 9]
 
       [b]
       preempt: True
+      
+      [b.nested]
       dict: {
           'a': 1,
           'b': %(answer)s,
-          'c': ['%(answer)s', %(answer)s]
+          'c': ['%(answer)s', '%(answer)s'],
         }
+      
+      [b.nested.nested-again]
+      movie: inception
       """
     )
-
-    self.ini2_content = textwrap.dedent(
+    ini2_content = dedent(
       """
       [a]
       fast: True
 
       [b]
       preempt: False
+      
+      [c.child]
```

Comment:
Note that there is no `c` scope defined. This one is added to ensure that we only report the section `c.child` but not `c`.